src/docs/release/breaking-changes/test-widgets-flutter-binding-clock.md

```diff
@@ -0,0 +1,104 @@
+---
+title: `TestWidgetsFlutterBinding.clock` change
+description: The `Clock` implementation will now come from package:clock.
+---
+
+## Summary
+
+The `TestWidgetsFlutterBinding.clock` will now come from `package:clock` and
```

Comment:
please adjust the tense so that if refers to the change as being in the past. Imagine the reader is a beginner in a year's time, dealing with Flutter for the first time working on a codebase they inherited.